examples/android/CHIPTool/app/src/main/java/com/google/chip/chiptool/clusterclient/SensorClientFragment.kt

```diff
@@ -165,13 +165,15 @@ class SensorClientFragment : Fragment() {
       val deviceId = binding.deviceIdEd.text.toString().toULong().toLong()
       val endpointId = binding.endpointIdEd.text.toString().toInt()
       val clusterName = binding.clusterNameSpinner.selectedItem.toString()
-      val clusterSubscribe = CLUSTERS[clusterName]!!["subscribe"] as (Long, Int, ReadCallback) -> Unit
+      val clusterId = CLUSTERS[clusterName]!!["clusterId"] as Long
+      val attributeId = CLUSTERS[clusterName]!!["attributeId"] as Long
       val device = ChipClient.getConnectedDevicePointer(requireContext(), deviceId)
-      val callback = makeReadCallback(clusterName, true)
+      val callback = makeReadCallback(clusterName, false)
```

Comment:
readability: "xxx=" false, what does false reprents?

Reply:
Sorry. It seems that the part that was false when tested was uploaded as it is.
When true, the graph is displayed; when false, the graph is not displayed.